google/auth/external_account.py

```diff
@@ -52,7 +52,7 @@
 # Cloud resource manager URL used to retrieve project information.
 _CLOUD_RESOURCE_MANAGER = "https://cloudresourcemanager.googleapis.com/v1/projects/"
 # Default Google sts token url.
-_DEFAULT_TOKEN_URL = "https://sts.googleapis.com/v1/token"
+_DEFAULT_TOKEN_URL = "https://sts.{universe_domain}/v1/token"
```

Comment:
This is fine for now but if the URL continues to get complex it should be wrapped in a type.

Reply:
SG, should not get more complex than this, but will make that change if it does.